source/backend/dal/Repositories/Interfaces/IEntityNoteRepository.cs

```diff
@@ -14,10 +14,14 @@ IEnumerable<T> GetAll<T>(Func<T, bool> predicate)
 
         IEnumerable<PimsNote> GetAllActivityNotesById(long entityId);
 
-        IEnumerable<PimsNote> GetAllAcquisitionNotesById(long entityId);
+        IEnumerable<PimsNote> GetAllAcquisitionNotesById(long acquisitionId);
```

Comment:
I like that you've made this more readable but kinda surprised you just targeted AcqNotes?

Reply:
Didn't want to update Activity related code :)

Reply:
lol, makes sense.